vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -30,7 +30,7 @@ $global:excludeHashComparisionListInAzureChinaCloud = @(
 
 # defaultContainerdPackageUrl refers to the stable containerd package used to pull and cache container images
 # Add cache for another containerd version which is not installed by default
```

Comment:
Please cache this new containerd version.

Reply:
I update the defaultContainerdPackageUrl and it is already cached. Should I cache the older version 1.6.21?

Reply:
Forgot that the url is directly used for v1.6